src/Bicep.Core.Samples/ExamplesTests.cs

```diff
@@ -122,7 +125,10 @@ public void ExampleIsValid(ExampleData example)
             var compilation = new Compilation(AzResourceTypeProvider.CreateWithAzTypes(), syntaxTreeGrouping);
             var emitter = new TemplateEmitter(compilation.GetEntrypointSemanticModel(), BicepTestConstants.DevAssemblyFileVersion);
 
-            foreach (var (syntaxTree, diagnostics) in compilation.GetAllDiagnosticsBySyntaxTree())
+            // quiet the linter diagnostics
+            var overrideConfig = new ConfigHelper().GetDisabledLinterConfig();
```

Comment:
Can you add an issue for removing this? Ideally the examples should showcase best-practices (no linter warnings).

Reply:
Will do.